kci-dev/kci-dev.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
python2 as been deprecated years ago
python2 shebang is "#!/usr/bin/env python2"

python is usually just a symlink to your distribution python version

Reply:
According to PEP 394, For Python script publishers:

"Older Linux distributions will provide a python command that refers to Python 2, and will likely not provide a python2 command."
"Some Linux distributions will not provide a python command at all by default, but will provide a python3 command by default."

In my case (Debian bookworm) python is not provided at all (unless i install python-is-python3), so it is preferable to point to python3 which is supposed to work for all.

Reply:
if we decide to do so, 
we should change all the python file not just this one file

Reply:
> we should change all the python file not just this one file

Good point, will change all.